vibrato/src/lib.rs

```diff
@@ -55,10 +56,12 @@ mod num;
 mod sentence;
 pub mod token;
 pub mod tokenizer;
-pub mod trainer;
 mod utils;
 
-#[cfg(test)]
+#[cfg(feature = "train")]
```

Comment:
```suggestion
#[cfg(feature = "train")]
#[cfg_attr(docsrs, doc(cfg(feature = "train")))]
```
I guess that it is sufficient to set `#[cfg_attr...` here instead of setting it on each struct under this module.